app/__init__.py

```diff
@@ -30,8 +32,11 @@ def init_app(testing=False):
 
     # Register within app context
     with app.app_context():
-        from . import routes
+        from .routes import products, categories, document
 
         db.create_all()
+        print(f"MongoDbs:  {client.list_database_names()}")
+        col = doc_db["products"]
+        print(f"Collection: {doc_db.list_collection_names()}")
```

Comment:
can we use the logging module instead of print? it turns out that python prints are not efficient at all + one place to define a logger object would be better.